site/docs/develop/kernel.md

```diff
@@ -27,10 +27,13 @@ current state of a table in a data lake. Whenever a table has changed via for ex
 a so-called commit operation instructs Nessie to record the new state in a Nessie commit, which
 carries the `Contents` object(s).
 
-`IcebergTable` contains the _current_ and _global_ pointer to Iceberg's table metadata plus the
-ID of the snapshot defined in the table metadata. Since Iceberg's table metadata manages information
-that must be consistent across all branches in Nessie, it is stored as so-called _global state_.
-The value of the snapshot-ID is stored per Nessie named reference (branch or tag).
+`IcebergTable` contains the _current_ pointer to Iceberg's table metadata plus the
+global ID's called _idGenerators_ that are defined in the table metadata. 
+Since Iceberg's table metadata manages information that must be consistent across all branches in Nessie, 
+required information is stored as so-called _global state_.
```

Comment:
`information` -> `some information`? AFAIK, Iceberg table metadata contains components that are global (such as ID generators) but also components that are managed by branch.